lib/web/users.go

```diff
@@ -161,6 +163,52 @@ func deleteUser(r *http.Request, params httprouter.Params, m userAPIGetter, user
 	return nil
 }
 
+type privilegeTokenRequest struct {
+	// SecondFactorToken is the totp code.
+	SecondFactorToken string `json:"secondFactorToken"`
+	// U2FSignResponse is u2f sign response for a u2f challenge.
+	U2FSignResponse *u2f.AuthenticateChallengeResponse `json:"u2fSignResponse"`
```

Comment:
Please add Webauthn support. Better yet if we could pivot some of the tests from using U2F to Webauthn, since the former is marked for removal soon ™️

Reply:
added webauthn, but since the front doesn't implement this yet, i'm going to leave the testing out to save me time

Reply:
Ack. I have plans, in then near future, to go through tests and pivot most of them from U2F to Webauthn, so deletion of U2F is easier when the time comes. (I do intend to keep happy-path-scenarios with U2F until then, ofc.)